networkit/test/test_linkprediction.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+import unittest
+import os
+import networkit as nk
+
+class TestLinkprediction(unittest.TestCase):
+
+	def setUp(self):
+		self.L = nk.readGraph("input/looptest1.gml", nk.Format.GML) #without self-loops
+	
+	def testAdamicAdarIndex(self):		
+		AI = nk.linkprediction.AdamicAdarIndex()
+		AI.setGraph(self.L)
+		self.assertEqual(AI.run(0,7), 0.0)		
+
+	def testAdjustedRandIndex(self):		
+		ARI = nk.linkprediction.AdjustedRandIndex()
+		ARI.setGraph(self.L)
+		self.assertAlmostEqual(ARI.run(0,7), -0.44, delta=0.1)	
+
+	def testAlgebraicDistanceIndex(self):		
+		ADI = nk.linkprediction.AlgebraicDistanceIndex(self.L, 5, 10)
+		ADI.setGraph(self.L)
+		ADI.preprocess()
+		self.assertAlmostEqual(ADI.run(0,7), 0.31, delta=0.5)	
```

Comment:
The delta seems quite big. Is this intended? (Same remark also for the following tests in the file)